### PR TITLE
EARTH-1088: search form in header

### PR DESCRIPTION
**Pull request by buttonwillowsix**
# READY FOR REVIEW

# Summary
- Hooked up work from Exygy by as much of the styling as possible, and conforming classes/ids to what is provided by the exposed filter from the Acquia Search page
- Hide the search on mobile/table sizes since it needs to go elsewhere to match comps.
- Does not include hiding the navbar, since attempts to do that via CSS failed.
- Not sure this is the most graceful way to do this all, so suggestions are welcomed!

Comps to compare against: https://projects.invisionapp.com/share/YSCSGALZJ#/screens

# Needed By (Date)
- Review requested as soon as possible. I'd like to get at least simple search up.

# Urgency
- Medium/High

# Steps to Test

1. Add the Exposed form: acquia_search-page block to the header
2. Add the  class navbar__search to the block (in block class field)
3. Remove the label on the exposed filter (Search)
4. Make sure the exposed filter on the form is set to NO reset button
5. Check that there is a) no button b)the search box expands and c) search box works.
6. Please do try this locally. But to look at this in action paired with display suite stuff: http://earthode99.prod.acquia-sites.com
Note: this searches _Prod_, not your local.

Verify that a search icon appears and when you click on it, it opens a box for entering your search query. 

Comps for search:
https://projects.invisionapp.com/share/YSCSGALZJ#/screens
Note: the box that opens matches the original comps, not Joe's 
https://projects.invisionapp.com/share/DMBA56W5U#/screens

Screen Shot 2019-08-28 at 10 31 52 AM
Screen Shot 2019-08-28 at 10 32 10 AM

# Affected Projects or Products
- Search on Stanford Earth

# Associated Issues and/or People
- EARTH-1088
https://stanfordits.atlassian.net/browse/EARTH-1088

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by cjwest** [comment]
@buttonwillowsix ,
As we walked through this together, it appears to be working as expected.
Do you want me to merge this, or do you want to get another review from Kassie or Mike?

**Comment by buttonwillowsix** [comment]
> @buttonwillowsix ,
> As we walked through this together, it appears to be working as expected.
> Do you want me to merge this, or do you want to get another review from Kassie or Mike?

@cjwest I mentioned this to Kassie in our meeting this morning and asked that she eyeball it. So let's wait for her review.

**Review by pookmish — APPROVED** [review]
all looks good and functioning as expected

**Pull request merged**
